composer.json

```diff
@@ -17,18 +17,20 @@
         }
     ],
     "require": {
-        "doctrine/orm": "^2.5.3",
         "php": "^7.2",
+        "doctrine/orm": "^2.5.3",
         "sulu/sulu": "^2.0.3 || ^2.1@dev",
         "symfony/config": "^4.3 || ^5.0",
         "symfony/console": "^4.3 || ^5.0",
         "symfony/dependency-injection": "^4.3 || ^5.0",
         "symfony/event-dispatcher": "^4.3 || ^5.0",
         "symfony/event-dispatcher-contracts": "^1.0 || ^2.0",
         "symfony/form": "^4.3 || ^5.0",
-        "symfony/validator": "^4.3 || ^5.0",
+        "symfony/mime": "^4.4 || ^5.1",
+        "symfony/mailer": "^4.4 || ^5.1",
```

Comment:
This should still be "dev" dependencies at current state mailer should not be shipped with sulu form bundle. When somebody want to switch to the mailer it should actively require it in there composer.json file.

Also it should be here `^4.4 || ^5.0`